Classes/CZ805MPickup.uc

```diff
@@ -3,7 +3,7 @@ class CZ805MPickup extends KFWeaponPickup;
 defaultproperties
 {
      Weight=6.000000
-     cost=3250
+     cost=3000
```

Comment:
Why this was needed? I set it to 3250 to make the price difference from other guns.

Reply:
I think the CZ805M should not get reload speed bonus with medic perks, medics currently reload it faster than commando while also having a 100% magazine size bonus compared to commando's 35%. In the event that the reload speed bonus is removed, the price of the weapon should be reduced to compensate, since most of the CZ805M's purpose is having high medic dart charge and low dart cost so the price was not reduced that much.

Reply:
I'll check the medic reload speed bonus after the merge and remove or change it if it will look OP.